adhocracy4/test/helpers.py

```diff
@@ -33,3 +37,29 @@ def redirect_target(response):
 def render_template(string, context=None):
     context = Context(context or {})
     return Template(string).render(context)
+
+
+@contextmanager
+def skip_background_mail():
```

Comment:
> i disagree as the skip_background_task context is required to test if emails are/would sent in other modules. see for example the test_report_api.py

Oh that was not obvious to me and I missed that point.

>  because i expect this to be a common pattern i think it is nice to have a context that is named explicitly [...]

Okay so it is basically a `force_sync_email_sending` context?

Towards the option of disabling that in test. Couldn't I instead test whether the email job was enqueued properly? I am a bit in favour of solutions that do not require another context manager, because a lot of our test already use a context manager (either `freeze_time` or `with_phase`) and I am not a huge fan of nesting them to much.